Makefile

```diff
@@ -22,13 +22,18 @@ LINUX_BIN := $(LINUX)/arch/riscv/boot/Image
 DEBIAN ?=
 ROOTFS_IMAGE := $(DEBIAN)/image.qcow2
 INITRD_IMAGE := $(DEBIAN)/initrd
+BUILDROOT_IMAGE := $(BUILDROOT)/output/images/rootfs.ext2
 
 RELEASE_BINS := target/riscv64gc-unknown-none-elf/release/
 DEBUG_BINS := target/riscv64gc-unknown-none-elf/debug/
 
 KERNEL_ADDR := 0xc0200000
 INITRD_ADDR := 0xc2200000
 BOOTARGS := keep_bootcon earlycon=sbi
+BUILDROOT_BOOTARGS := root=/dev/nvme0n1 console=hvc0 earlycon=sbi
```

Comment:
these bootargs should work for debian too, right?

Reply:
The console & earlycon part are same. The root part differs as the distro may not have the rootfs on the first partition. I can modify BUILDROOT_BOOTARGS to use $BOOTARGS and have "root=" specific to rootfs type.